tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -228,6 +229,16 @@ struct RNTupleLocator {
    }
 };
 
+/// Used to specify the underlying RNTuples in RNTupleProcessor and RNTupleReader::OpenFriends()
+struct RNTupleOpenSpec {
+   std::string fNTupleName;
+   std::string fStorage;
+   RNTupleReadOptions fOptions;
+
+   RNTupleOpenSpec() = default;
```

Comment:
Also add rule of five methods

Reply:
I have to correct myself here, the default constructor is not involved with the rule of five, so actually it would be better to not write any other of the special methods in the class. Apologies for the confusion!